x/blob/client/cli/payforblob.go

```diff
@@ -81,17 +84,16 @@ func CmdPayForBlob() *cobra.Command {
 				return err
 			}
 
-			// If there is a file path input we'll check for the file extension
 			if path != "" {
-				if filepath.Ext(path) != ".json" {
-					return fmt.Errorf("invalid file extension, require json got %s", filepath.Ext(path))
+				if filepath.Ext(path) != FileInputExtension {
```

Comment:
[Question] Is there a check in the code for whether the file exists in the specified path (hence showing proper error if not)?

Reply:
Looks like yes.
```
$ ./build/celestia-appd tx blob pay-for-blob --input-file foo.json --from validator
Error: open foo.json: no such file or directory
```